lib/ui/views/messenger/layouts/create_talk_sheet.dart

```diff
@@ -37,39 +67,94 @@ class CreateTalkSheet extends ConsumerWidget {
-                              )),
+                            onPressed: () {
+                              Sheets.showAppHeightNineSheet(
+                                context: context,
+                                ref: ref,
+                                widget: const CreateGroupSheet(),
+                              );
+                            },
+                            child: Row(
+                              children: [
+                                Icon(
+                                  Icons.group_add_outlined,
+                                  color: theme.textStyleSize14W700Primary.color,
+                                ),
+                                const SizedBox(
+                                  width: 8,
+                                ),
+                                Text(
+                                  localizations.newGroup,
+                                  style: theme.textStyleSize14W700Primary,
+                                ),
+                              ],
+                            ),
+                          ),
+                          Divider(
+                            color: theme.textStyleSize14W700Primary.color,
+                          ),
+                          TextButton(
+                            onPressed: () {
+                              Sheets.showAppHeightNineSheet(
+                                context: context,
+                                ref: ref,
+                                widget: const AddContactSheet(),
+                              );
+                            },
+                            child: Row(
+                              children: [
+                                Icon(
+                                  Iconsax.user_add,
+                                  color: theme.textStyleSize14W700Primary.color,
+                                ),
+                                const SizedBox(
+                                  width: 8,
+                                ),
+                                Text(
+                                  localizations.newContact,
+                                  style: theme.textStyleSize14W700Primary,
+                                ),
+                              ],
+                            ),
+                          ),
                         ],
                       ),
+                      Divider(color: theme.textStyleSize14W700Primary.color),
                       const SizedBox(
                         height: 15,
                       ),
+                      Text(
+                        localizations.contactsHeader,
+                        style: theme.textStyleSize14W200Primary
+                            .copyWith(fontWeight: FontWeight.bold),
+                      ),
+                      const SizedBox(
+                        height: 8,
+                      ),
+                      Visibility(
```

Comment:
See the bottom of this article : https://protocoderspoint.com/different-between-visibility-opacity-offstage-flutter-show-hide-widgets/?expand_article=1

Reply:
For my personal usage and experience, I prefer use Visibility instead of conditions because I don't like to use such logic in widgets. I don't know what's your point of view for this project thought.

Reply:
In complex screens, with conditions within listview cells, the use of if can help for performance reason. In this very simple screen, I doubt that this is mandatory. It's up to you to help me if you prefer the if usage or not.

Reply:
ok for this PR. It could be interesting to discuss this (minor) point with @Chralu